cbor/codec.go

```diff
@@ -0,0 +1,29 @@
+package cbor
+
+import "github.com/fxamacker/cbor/v2"
+
+// CBORCodec encode decode
+type CBORCodec struct {
```

Comment:
stuttering by repeating CBOR is discouraged - why not just Codec. USer code will refer to this as cbor.Codec which is clear enough?